circuits/prover/src/lib.rs

```diff
@@ -1 +1,198 @@
+pub mod rapidsnark;
+pub mod traits;
```

Comment:
This should probably be private, and then we just re-export the actual trait inside. `traits` is too a generic kind of naming.